documentation/modules/configuring/ref-storage-ephemeral.adoc

```diff
@@ -5,14 +5,26 @@
 [id='ref-ephemeral-storage-{context}']
 = Ephemeral storage
 
+[role="_abstract"]
+Ephemeral data storage is transient. 
+All pods on a node share a local ephemeral storage space. 
+Data is retained for as long as the pod that uses it is running.
+The data is lost when a pod restarts. 
+Although a pod can recover data in a highly available environment.  
+
+Because of its transient nature, ephemeral storage is not recommended for production.
+
 Ephemeral storage uses `{K8sEmptyDir}` volumes to store data.
-To use ephemeral storage, set the `type` field to `ephemeral`.
+An `emptyDir` volume is created when a pod is assigned to a node. 
+You can set the total amount of storage for the `emptyDir` using the `sizeLimit` property .
 
-IMPORTANT: `emptyDir` volumes are not persistent and the data stored in them is lost when the pod is restarted.
+IMPORTANT: `emptyDir` volumes are not persistent and the data stored in them is lost when a pod is restarted.
 After the new pod is started, it must recover all data from the other nodes of the cluster.
 Ephemeral storage is not suitable for use with single-node ZooKeeper clusters or for Kafka topics with a replication factor of 1. This configuration will cause data loss.
```

Comment:
It seems like we basically repeat the content here within a 15 lines of text. The `IMPORTANT` block basically says the same as bit chunk of the first paragraph. MAybe we can simplify it and keep only the `IMPORTANT` part or something like that?

Reply:
I just retained the specific configuration info in the IMPORTANT note. I think the general stuff is better in the intro.